src/lib/dev-environment/dev-environment-cli.js

```diff
@@ -322,9 +333,11 @@ async function processComponent( component: string, preselectedValue: string, de
 
 		if ( isPathValid ) {
 			break;
-		} else {
+		} else if ( isStdinTTY ) {
```

Comment:
We need to prevent infinite loops if the initial value is somehow invalid, because it will be impossible to change it:

```
Warning: Provided path "/home/volodymyr/work/vip-cli/dist/lib/dev-environment" is missing following files/folders: languages, plugins, themes, private, images, client-mu-plugins, vip-config. Learn more: https://docs.wpvip.com/technical-references/vip-codebase/#1-wordpress
Warning: Provided path "/home/volodymyr/work/vip-cli/dist/lib/dev-environment" is missing following files/folders: languages, plugins, themes, private, images, client-mu-plugins, vip-config. Learn more: https://docs.wpvip.com/technical-references/vip-codebase/#1-wordpress
Warning: Provided path "/home/volodymyr/work/vip-cli/dist/lib/dev-environment" is missing following files/folders: languages, plugins, themes, private, images, client-mu-plugins, vip-config. Learn more: https://docs.wpvip.com/technical-references/vip-codebase/#1-wordpress
...
```